js/dav/dav.js

```diff
@@ -3181,20 +3182,21 @@ var OAuth2 = (function (_Transport2) {
               xhr.open(request.method, url, true /* async */);
               xhr.setRequestHeader('Authorization', 'Bearer ' + token);
               if (transformRequest) transformRequest(xhr);
-              context$3$0.next = 17;
+              xhr.setRequestHeader('requesttoken', oc_requesttoken);
```

Comment:
There is an additional `setRequestHeader` because of https://github.com/owncloud/contactsrework/pull/43/files#diff-620b1e6c50bc4d39c4e310a6f153e580R94